routes/licenses.js

```diff
@@ -43,7 +47,12 @@ routes.push({
     const { licenseStore } = request.server.app.services;
     const { licenseId } = request.params;
     const licenses = licenseStore.compatible(licenseId);
-    const response = licenses.map(({ url, name }) => ({ url: encodeURI(url), code: name }));
+    const response = licenses.map(({ id: code, url, name, groups }) => ({
+      code,
+      name,
+      url,
+      groups,
+    }));
```

Comment:
Looks like it'd make sense to extract the mapping function into a formatter/serializer and use it in the other route handlers here as well to keep responses consistent 😉